java/src/org/certificatetransparency/ctlog/LogEntry.java

```diff
@@ -0,0 +1,12 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class LogEntry {
+  public Ct.X509ChainEntry x509Entry;
+  public Ct.PrecertChainEntry preCertEntry;
```

Comment:
Except for `PreCert`?

The former protobuf setter was called `setPrecertEntry`, so you mean that this should be called `precertEntry`?